kolibri/plugins/management/assets/src/views/class-enroll-page/index.vue

```diff
@@ -324,24 +346,21 @@
 
   @require '~kolibri.styles.definitions'
 
-  $toolbar-height = 36px
+  $table-row-selected = #e0e0e0
+  $table-row-hover = #eee
```

Comment:
slight smell here of introducing new color values that aren't used elsewhere in the app. probably okay for now, but would be better if we were re-using existing values from core-theme.styl in case we ever e.g. want to do white text on dark background